### PR TITLE
Fix and update bind test

### PR DESCRIPTION
**Pull request by dzedro**
- Related ticket: https://progress.opensuse.org/issues/159465
  https://gitlab.suse.de/qa-maintenance/qam-openqa-yml/-/merge_requests/743
- Verification run:
https://openqa.suse.de/tests/overview?version=15-SP6&build=jpupava_bind&distri=sle 15-SP6
https://openqa.suse.de/tests/overview?distri=sle&version=15-SP5&build=20240507-1&groupid=414 15-SP5
https://openqa.suse.de/tests/overview?distri=sle&version=15-SP4&build=20240507-1&groupid=414 15-SP4

### CODE REVIEW

File: tests/console/bind.pm [hunk 1]
```diff
@@ -41,15 +42,19 @@ sub run {
     elsif (is_sle('>=15')) {
         # bind-utils for dig, net-tools-deprecated for ifconfig, perl-IO-Socket-INET6 for reclimit,
         # perl-Net-DNS for xfer, dnspython for chain test
-        zypper_call 'in bind rpm-build bind-utils net-tools-deprecated perl-IO-Socket-INET6 perl-Socket6 perl-Net-DNS python3-dnspython';
+        zypper_call 'in bind rpm-build bind-utils net-tools-deprecated perl-IO-Socket-INET6 perl-Socket6 perl-Net-DNS python3-dnspython git-core python3-pytest python3-hypothesis jemalloc-devel libcmocka-devel';
     }
     # enable source repositories to get latest source packages
```
[the remark this answers is not in the document]
Comment (dzedro) [reply tests/console/bind.pm:47]: You mean like is enabled below ?

[the remark this answers is not in the document]
Comment (dzedro) [reply tests/console/bind.pm:47]: source packages are only in source repos AFAIK

[the remark this answers is not in the document]
Comment (rfan1) [reply tests/console/bind.pm:47]: Yes, based on  https://bugzilla.suse.com/show_bug.cgi?id=1223332,  the src packages are located at source repo before GMC, once the os relased. the newer update packages and src packages are both located at updates repo. anyway, it is not a big deal.

For sle15sp6, enable the srouce repo is required.

[the remark this answers is not in the document]
Comment (rfan1) [reply tests/console/bind.pm:47]:
```
git-host:~ # zypper se -s bind|grep srcpackage
  | bind                                        | srcpackage | 9.16.48-150500.8.18.1                     | noarch | SLE-Module-Basesystem15-SP5-Updates
  | bind                                        | srcpackage | 9.16.48-150500.8.18.1                     | noarch | SLE-Module-Server-Applications15-SP5-Updates
  | bind                                        | srcpackage | 9.16.48-150500.8.16.1                     | noarch | SLE-Module-Basesystem15-SP5-Updates
  | bind                                        | srcpackage | 9.16.48-150500.8.16.1                     | noarch | SLE-Module-Server-Applications15-SP5-Updates
  | bind                                        | srcpackage | 9.16.44-150500.8.12.2                     | noarch | SLE-Module-Basesystem15-SP5-Updates
  | bind                                        | srcpackage | 9.16.44-150500.8.12.2                     | noarch | SLE-Module-Server-Applications15-SP5-Updates
  | bind                                        | srcpackage | 9.16.42-150500.8.7.1                      | noarch | SLE-Module-Basesystem15-SP5-Updates
  | bind                                        | srcpackage | 9.16.42-150500.8.7.1                      | noarch | SLE-Module-Server-Applications15-SP5-Updates
  | bind                                        | srcpackage | 9.16.42-150500.8.3.1                      | noarch | SLE-Module-Basesystem15-SP5-Updates
  | bind                                        | srcpackage | 9.16.42-150500.8.3.1                      | noarch | SLE-Module-Server-Applications15-SP5-Updates
```

**Comment by rfan1** [comment]
Hello @dzedro https://openqa.suse.de/tests/14250094#step/bind/31 the bind test fails on sle15sp2/p3

**Comment by dzedro** [comment]
> Hello @dzedro https://openqa.suse.de/tests/14250094#step/bind/31 the bind test fails on sle15sp2/p3

yes, I know I softfailed previous job but openQA restarted it and failed.

**Pull request merged**